### ISSUE TITLE
mplayer always crash: Could not map image: Out of memory

### ISSUE THREAD

**Issue by Giova84** [haikuports/haikuports#1978]
I installed, on Haiku gcc2h (hrev51714) the new mplayer (mplayer_x86-1.3.0-5-x86_gcc2.hpkg) package. Whenever I attempt to play a media file, I always get: `runtime_loader: /boot/system/bin/mplayer: Could not map image: Out of memory`.
On this  Desktop PC I have enough memory: 6 GiB of DDR3 RAM.

I attach the debog output:
https://pastebin.com/M7aGa6Qs

**Comment by Giova84** [comment]
For the sake of completeness: here I have `mplayer_x86-1.1.4.8.3-5-x86_gcc2.hpkg` which I built by myself a long time ago, which still works without any issue.

**Comment by extrowerk** [comment]
Does it still happens?

**Comment by kenmays** [comment]
Yes, the issue is still revelant. Tested on hrev52080 x86_gcc2 w/ mplayer_x86-1.3.0-8.

**Comment by humdingerb** [comment]
I see the same issue with mplayer_x86-1.4.0-9-x86_gcc2.hpkg on hrev53872.
It's showing `runtime_loader: /boot/system/bin/mplayer: Could not map image: Out of memory` even when just calling "mplayer --help"...

Does it work on 64bit? Should we mark the 32bit version as broken in the recipe?

**Comment by extrowerk** [comment]
64 bit works for me.

**Comment by kenmays** [comment]
@humdingerb - Yes, verified as broken on x86_gcc2 (mplayer_x86-1.4.0-8) using hrev53874 x86_gcc2.. Crash report included.

[mplayer-747-debug-16-02-2020-16-49-13.txt](https://github.com/haikuports/haikuports/files/4210295/mplayer-747-debug-16-02-2020-16-49-13.txt)
